src/components/custom/_signup-affiliate-details.js

```diff
@@ -78,7 +64,7 @@ const SignupButton = styled(Button)`
         font-size: 1.75rem;
     }
 `
-const LoginText = styled(Text)`
+const LoginText = styled(Header)`
```

Comment:
This can be achieved with `<Header type="paragraph-1 mt="1.6rem" tabletL={{ mt: '3.75rem' }}></Header>`
Although the 3.75rem should be in px for responsive view.

Reply:
Updated props with style values here, thanks.